netmap/netmap_contract.go

```diff
@@ -94,6 +94,24 @@ func _deploy(data interface{}, isUpdate bool) {
 
 	if isUpdate {
 		common.CheckVersion(args.version)
+
+		prefix := []byte(snapshotKeyPrefix)
+		for i := 0; i < DefaultSnapshotCount; i++ {
+			key := append(prefix, byte(i))
+			data := storage.Get(ctx, key)
+			if data != nil {
+				nodes := std.Deserialize(data.([]byte)).([]Node)
```

Comment:
Can we decode to list of structures with old format in order to not loose the state?

Reply:
I think it is ok now -- the only new state is maintenance and it is added in this release, so all ready stored nodes are `Online`.